commit 4d2e9c98b5b886d80fb7396aef89ca50b0d5fcfe
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Sep 27 23:33:29 2009 +0200

    Use #!/bin/bash instead of #!/bin/sh
    
    The script makes big use of bashisms.

---
 sutils/MAKEDEV.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sutils/MAKEDEV.sh
+++ b/sutils/MAKEDEV.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Make standard devices
 #
